<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tke_form_print_poshtu_dlya_pilgovykiv</class>
 <widget class="QWidget" name="tke_form_print_poshtu_dlya_pilgovykiv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Друк попереджувальної пошти для пільговиків</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tke/tke_rc">
    <normaloff>:/pictures/pictures/printer.png</normaloff>:/pictures/pictures/printer.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetPilgovyky">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Рахунок</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ПІБ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Адреса</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Катег. пільги</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Здано док.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Друкувати</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_reload">
       <property name="text">
        <string>Поновити список</string>
       </property>
       <property name="icon">
        <iconset resource="tke/tke_rc">
         <normaloff>:/pictures/pictures/redeal.png</normaloff>:/pictures/pictures/redeal.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_selectAll">
       <property name="text">
        <string>Позначити усіх пільговиків</string>
       </property>
       <property name="icon">
        <iconset resource="tke/tke_rc">
         <normaloff>:/pictures/pictures/button_ok.png</normaloff>:/pictures/pictures/button_ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_deselectAll">
       <property name="text">
        <string>Зняти позначення усіх пільговиків</string>
       </property>
       <property name="icon">
        <iconset resource="tke/tke_rc">
         <normaloff>:/pictures/pictures/button_cancel.png</normaloff>:/pictures/pictures/button_cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>261</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_printSelected">
       <property name="text">
        <string>Друкувати позначені</string>
       </property>
       <property name="icon">
        <iconset resource="tke/tke_rc">
         <normaloff>:/pictures/pictures/printer_default.png</normaloff>:/pictures/pictures/printer_default.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_printSelectedOnPrinter">
       <property name="text">
        <string>Друкувати на принтері ...</string>
       </property>
       <property name="icon">
        <iconset resource="tke/tke_rc">
         <normaloff>:/pictures/pictures/printer_conf.png</normaloff>:/pictures/pictures/printer_conf.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidgetPilgovyky</tabstop>
  <tabstop>pushButton_selectAll</tabstop>
  <tabstop>pushButton_deselectAll</tabstop>
  <tabstop>pushButton_printSelected</tabstop>
  <tabstop>pushButton_printSelectedOnPrinter</tabstop>
 </tabstops>
 <resources>
  <include location="tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
